### PR TITLE
update systemd.markdown

### PR DESCRIPTION
**Pull request by jbob762**
Used this page to get hass set up on my Pi and thought the changes above might help people new to hass figure out which template to use. Also had trouble getting the command to run with sudo, added a workaround.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10094"></a>

### CODE REVIEW

File: source/_docs/autostart/systemd.markdown [hunk 1]
```diff
@@ -35,7 +36,7 @@ WantedBy=multi-user.target
 
 ### Python virtual environment
 
-If you've setup Home Assistant in `virtualenv` following our [Python installation guide](/getting-started/installation-virtualenv/) or [manual installation guide for Raspberry Pi](/getting-started/installation-raspberry-pi/), the following template should work for you. If Home Assistant install is not located at `/srv/homeassistant`, please modify the `ExecStart=` line appropriately. `YOUR_USER` should be replaced by the user account that Home Assistant will run as (e.g `homeassistant`).
+If you've setup Home Assistant in `virtualenv`, the following template should work for you.If Home Assistant install is not located at `/srv/homeassistant`, please modify the `ExecStart=` line appropriately. `YOUR_USER` should be replaced by the user account that Home Assistant will run as (e.g `homeassistant`).
```
Comment (DubhAd) [inline source/_docs/autostart/systemd.markdown:39]: Not sure why you deleted the space.

Only the Raspberry Pi guide gives the location of `/srv/homeassistant`, so this edit could cause confusion. Those following the first guide will likely have installed it in their home directory.

**Review by DubhAd — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
A small and friendly notification.

Are you planning on moving forward on this PR? 
I didn't see any activity recently, while there are still things to do. I hope you consider picking it up soon! 👍

**Comment by jbob762** [comment]
Sorry, new to this. What else do I need to do?

Thanks

**Comment by frenck** [comment]
Well, @jbob762, if you look at the PR page, you'd see DubhAd left a comment...

File: source/_docs/autostart/systemd.markdown (hunk 1 above)
Comment (jbob762) [reply source/_docs/autostart/systemd.markdown:39]: Sounds good. Don’t know what happened to the space.

**Comment by frenck** [comment]
Since there was no response at all, I'm declaring this PR stale and closing it.
Feel free to re-open this PR when you feel like that is the appropriate thing to do.

**Pull request closed without merging**